tests/cancun/eip4844_blobs/test_excess_blob_gas.py

```diff
@@ -457,9 +457,9 @@ def all_invalid_blob_gas_used_combinations() -> Iterator[Tuple[int, int]]:
     Returns all invalid blob gas used combinations.
     """
     for new_blobs in range(0, SpecHelpers.max_blobs_per_block() + 1):
-        for header_blob_gas_used in range(0, SpecHelpers.max_blobs_per_block() + 1):
-            if new_blobs != header_blob_gas_used:
-                yield (new_blobs, header_blob_gas_used * Spec.GAS_PER_BLOB)
+        for header_blobs in range(0, SpecHelpers.max_blobs_per_block() + 1):
```

Comment:
I think this change is not entirely necessary, and we are refactoring precisely this code in another PR for EIP-7691+EIP-7762, it's going to cause an unnecessary conflict when rebasing.